optihood/entities.py

```diff
@@ -150,11 +150,14 @@ class ProfileLabels(_enum.StrEnum):
 
 class ProfileTypes(_enum.StrEnum):
     demand = "demand_profiles"  # mandatory
+    demandProfiles = 'demandProfiles'  # because both implementations exist atm
```

Comment:
I'd add a TODO here to clean this up when we move completely to the "demand_profiles" convention